contentcuration/contentcuration/frontend/shared/views/TipTapEditor/TipTapEditor/components/image/ImageUploadModal.vue

```diff
@@ -212,6 +212,13 @@
       const imageProcessor = inject('imageProcessor', {});
       const { processFile, ACCEPTED_MIME_TYPES } = imageProcessor;
 
+      const supportedFileTypesText = computed(() => {
+        const extensions = ACCEPTED_MIME_TYPES.map(type =>
+          type.replace('image/', '').replace('svg+xml', 'svg'),
+        );
+        return supportedFileTypes$() + extensions.join(', ');
```

Comment:
You would then pass this like this:

```suggestion
        return supportedFileTypes$({ extensions: extensions.join(', ') });
```